airbyte-ci/connectors/pipelines/pipelines/helpers/cli.py

```diff
@@ -93,7 +87,7 @@ async def invoke_commands_sequentially(ctx: click.Context, commands: List[click.
     """
     command_executions_results = []
     for command in commands:
-        command_executions_results.append(await ctx.invoke(command))
+        command_executions_results.append(await command.invoke(ctx))
```

Comment:
Confusing swap 😄

Reply:
I know. But otherwise I get an exception that the invoked command does not have a callback...